### ISSUE TITLE
No help to configure the right JDK

### ISSUE THREAD

**Issue by linuswagner** [usethesource/rascal-language-servers#430]
**Describe the bug**

When trying to launch a Rascal shell in VS Code on a machine where the right JDK is installed but not configured correctly for Rascal to find, I encounter the `Failed to start the Rascal REPL, check Rascal Output Window - Error: User setup required for jdk 11`.

The documentation links to [this](https://www.rascal-mpl.org/docs/GettingStarted/CreateNewProject/). However, there is no help provided to correctly set up an existing JDK on my local machine (I'd assume, the plugin checks for the `JAVA_HOME` environment variable?).

**Expected behavior**
Get instructions how to correctly set up an existing JDK so that Rascal can discover it.

**Screenshots**
<img width="255" alt="Screenshot 2024-08-26 at 18 04 31" src="https://github.com/user-attachments/assets/eff222ac-5a63-4ae2-a9ff-978b109d25df">

**Desktop (please complete the following information):**
- VS Code plugin `v0.11.2`

**Comment by DavyLandman** [comment]
Yes, the custom JDK setting is not implemented yet, you're the first to notice that in a year ;) Could you give some information on how you got in this path? I assume the java detection noticed no installed java, and asked you for a way to solve it?

If you want, you can make a PR that adds the setting to the extension? 

For context, normally we check regular places where java is defined (such as `JAVA_HOME` environment, `java` on the `PATH`, and our own `~/.jvm` folder). Otherwise we offer the user to download java for them (and also keep it up to date).
